packages/jest-circus/src/utils.ts

```diff
@@ -419,6 +425,10 @@ const _getError = (
   return asyncError;
 };
 
+export const isError = (e: unknown): e is Error => {
+  return String(e) === '[object Error]' || e instanceof Error;
+};
+
```

Comment:
I added this very early on and it may not longer be used anywhere. Consider removing.